rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec

```diff
@@ -0,0 +1,50 @@
+%{?scl:%scl_package python-billiard}
+%{!?scl:%global pkg_name %{name}}
+
+%global srcname billiard
+
+Name:		%{?scl_prefix}python-billiard
+Version:	3.3.0.23
+Release:	1%{?dist}
+Summary:	Multiprocessing Pool Extensions
+
+License:	BSD
+URL:		http://pypi.python.org/pypi/billiard
+Source0:	http://pypi.python.org/packages/source/b/%{srcname}/%{srcname}-%{version}.tar.gz
+
+BuildRequires:	%{?scl_prefix_python}python-devel
+BuildRequires:	%{?scl_prefix_python}python-setuptools
+BuildRequires:	%{?scl_prefix_python}python-nose
+%{?scl:BuildRequires:	%{scl}-build %{scl}-runtime}
+
+%{?scl:Requires:	%{scl}-runtime}
+
+BuildRoot:      %{_tmppath}/%{pkg_name}-%{version}-%{release}-root-%(%{__id_u} -n)
```

Comment:
Correct, we don't.  I removed the overridden build root lines from other pakcages, it appears I left it in here